src/runtime_src/core/edge/user/aie/aie.h

```diff
@@ -77,8 +87,16 @@ class Aie {
     XAieGbl_Config *aieConfigPtr; // AIE configuration pointer
     XAieGbl aieInst;              // AIE global instance
     XAieGbl_HwCfg aieConfig;      // AIE configuration pointer
+
+    void
+    submit_sync_bo(uint64_t paddr, std::vector<gmio_type>::iterator& gmio, enum xclBOSyncDirection dir, size_t size);
+
+    /* Wait for all the BD trasters for a given channel */
```

Comment:
I am so happy to see a comment! \o/
What is "trasters"? Some secret piece of hardware?
Please feel free to add a lot of comments.
I love open-source code and even more READABLE open-source code... :-)

Reply:
Thanks for catching this.